Sources/Datadog/RUM/Debugging/RUMDebugging.swift

```diff
@@ -108,7 +108,7 @@ internal class RUMDebugging {
             canvas.addSubview(view)
         }
         if canvas.superview == nil,
-            let someWindow = UIApplication.managedShared?.keyWindow {
+            let someWindow = UIApplication.shared.windows.filter {$0.isKeyWindow}.first {
```

Comment:
Thanks for fixing the warning!

I think we should fallback to `managedShared` here. Also linter is going to probably complain.

Can you please run `sh tools/lint/run-linter.sh`?

Reply:
done!